invenio_app_rdm/records_ui/views/deposits.py

```diff
@@ -71,10 +61,14 @@ def get_form_pids_config():
                                    "field blank to have one automatically "
                                    "assigned when publishing.")
             .format(scheme_label=scheme_label),
-            "unmanaged_help_text": _("Copy and paste here your {scheme_label}.")  # noqa
+            "unmanaged_help_text": _("Copy and paste here your {scheme_label} "
+                                     "or leave this field blank to have one "
```

Comment:
Quick question: didn't we agree that when:
- Chossing "external" i.e. the radio button is in "yes"
- And leaving the text blank

It would send:
```
"pids": {
    "doi": {
         "provider": "external"
         "identifier": ""    # or without this key anyway
     }
 }
 ```
 
This would not result in a new DOI, this would result in a validation error.